src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -47,13 +89,31 @@ public void SetAccessControl_DirectoryInfo_DirectorySecurity_InvalidArguments()
         }
 
         [Fact]
+        public void SetAccessControl_DirectoryInfo_DirectorySecurity()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+            DirectorySecurity directorySecurity = new DirectorySecurity();
+
+            FileSystemAclExtensions.SetAccessControl(directoryInfo, directorySecurity);
```

Comment:
@the-dwyer, not necessarily; it might be sufficient to just do so on a directory other than the current one, which is what this appears to currently be doing, e.g. creating a temporary directory and setting permissions on that.  @ianhays may have other/better suggestions.